.github/workflows/build.yml

```diff
@@ -4,67 +4,30 @@ on: [push, pull_request]
 
 jobs:
   test:
-    name: Test ${{ matrix.os }} / ${{ matrix.toxenv }}
+    name: Test ${{ matrix.os }}-python${{ matrix.python-version }}
     runs-on: ${{ matrix.os }}
     strategy:
       fail-fast: false
       matrix:
-        include:
-          - {os: windows-latest, python_Version: '3.11', toxenv: 'py311'}
-          - {os: windows-latest, python_Version: '3.10', toxenv: 'py310'}
-          - {os: windows-latest, python_Version: '3.8', toxenv: 'py38'}
-          - {os: windows-latest, python_Version: '3.9', toxenv: 'py39'}
-          - {os: ubuntu-latest, python_Version: '3.11', toxenv: 'py311'}
-          - {os: ubuntu-latest, python_Version: '3.10', toxenv: 'py310'}
-          - {os: ubuntu-latest, python_Version: '3.8', toxenv: 'py38'}
-          - {os: ubuntu-latest, python_Version: '3.9', toxenv: 'py39'}
-          - {os: macos-latest, python_Version: '3.11', toxenv: 'py311'}
-          - {os: macos-latest, python_Version: '3.10', toxenv: 'py310'}
-          - {os: macos-latest, python_Version: '3.8', toxenv: 'py38'}
-          # missing numcodecs wheels on 3.9. conda not yet an option. see gh-51
-          # {os: macos-latest, python_Version: '3.9', toxenv: 'py39'}
-
+        python-version: ['3.8', '3.9', '3.10', '3.11', '3.12']
+        os: ['windows-latest', 'macos-latest', 'ubuntu-latest']
     steps:
-      - uses: actions/checkout@v3
+      - uses: actions/checkout@v4
 
-      - uses: actions/setup-python@v4
-        name: Install Python ${{ matrix.python_version }}
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v4
         with:
-          python-version: ${{ matrix.python_version }}
-
-      - name: Install ubuntu libraries
```

Comment:
I'm concerned that by skipping these we're now omitting a test. Or did those tests get moved to napari-ome-zarr, @will-moore?